lib/OpenQA/Utils.pm

```diff
@@ -652,19 +652,17 @@ sub parse_tags_from_comments ($group, $res) {
 }
 
 sub detect_current_version ($path) {
-    # Get application version
     my $current_version = undef;
-    my $changelog_file = path($path, 'public', 'Changelog');
+    my $version_file = $path->child('version.txt');
+    if (-e $version_file) {
```

Comment:
Since you go with `version.txt` the `$current_version` should be always available, unless it is an older openQA version. So I believe the condition needs something more.